src/DocumentFormat.OpenXml/OpenXmlPartReader.cs

```diff
@@ -70,7 +70,23 @@ public OpenXmlPartReader(OpenXmlPart openXmlPart, bool readMiscNodes)
                 throw new ArgumentNullException(nameof(openXmlPart));
             }
 
-            _xmlReader = CreateReader(openXmlPart.GetStream(FileMode.Open), true, openXmlPart.MaxCharactersInPart, out _standalone, out _encoding);
+            _xmlReader = CreateReader(openXmlPart.GetStream(FileMode.Open), true, openXmlPart.MaxCharactersInPart, ignoreWhitespace: false, out _standalone, out _encoding);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the OpenXmlPartReader class using the supplied OpenXmlPart and Boolean values.
+        /// </summary>
+        /// <param name="openXmlPart">The OpenXmlPart to read.</param>
+        /// <param name="readMiscNodes">Specify false to indicate to the reader to skip all miscellaneous nodes. The default value is false.</param>
```

Comment:
both this and the following comment reference a default value. There are none that I can see.

Reply:
Updated the comment